test/requests/path_completion_test.rb

```diff
@@ -160,17 +160,33 @@ def with_file_structure(&block)
       # |       |-- baz.rb
       # |       |-- quux.rb
       FileUtils.mkdir_p(tmpdir + "/foo/support")
-      FileUtils.touch([
+      files = [
         tmpdir + "/foo/bar.rb",
         tmpdir + "/foo/baz.rb",
         tmpdir + "/foo/quux.rb",
         tmpdir + "/foo/support/bar.rb",
         tmpdir + "/foo/support/baz.rb",
         tmpdir + "/foo/support/quux.rb",
-      ])
+      ]
+
+      files.each do |file|
+        FileUtils.touch(file)
+        RubyLsp::Index.instance.synchronize([{
+          uri: "file://#{file}",
+          type: RubyLsp::Constant::FileChangeType::CREATED,
+        }])
+      end
```

Comment:
Would this be better as:
```suggestion
      changes = files.map do |file|
        FileUtils.touch(file)

        {
          uri: "file://#{file}",
          type: RubyLsp::Constant::FileChangeType::CREATED,
        }
      end
      RubyLsp::Index.instance.synchronize(changes)
```